letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -52,6 +52,9 @@ elif bashio::config.exists 'dns.google_creds'; then
 elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-google-domains" ]; then
     bashio::config.require 'dns.google_domains_access_token'
     PROVIDER_ARGUMENTS+=("--authenticator" "${DNS_PROVIDER}" "--${DNS_PROVIDER}-credentials" "/data/dnsapikey" "--${DNS_PROVIDER}-propagation-seconds" "${PROPAGATION_SECONDS}")
+#Namecheap
+elif bashio::config.exists 'dns.namecheap_username' && bashio::config.exists 'dns.namecheap_api_key'; then
```

Comment:
```suggestion
elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-namecheap" ]; then
    bashio::config.require 'dns.namecheap_username'
    bashio::config.require 'dns.namecheap_api_key'
```
I know you probably copied what netcup did but lets borrow the model from some of the others. If the user choose namecheap lets make it fail if they missed a required value. Rather then continuing down the if statements and failing on the else and then failing when required auth info is missing.

Reply:
Makes sense to me, updates made 👍